.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,26 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install --quiet codecov wheel flake8
```

Comment:
Would it make sense to use the `requirements.txt.in`/`requirements.txt` pattern we had discussed for this repo?

Reply:
This repository contains a legacy package in maintenance mode. It has multiple issues and I don't intend for us to use it in the long term. Instead our long term strategy is to deprecate this package and replace it with an industry-supported workflow management system. The only reason I'm adding the linter and fixing the errors in this PR is to be able to sanity check the changes that are necessary to add to this package in the short term for maintenance purposes.

With that in mind, no, further investment in the maintenance infrastructure of this package is not needed.